app/views/entities/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="bg-main flex justify-between text-white items-center h-12">
+  <%= link_to "<", group_entities_path(@group_id), class: 'text-xl mx-2'  %>
+  <h1 class="font-proxima-bold">ENTITIES</h1>
+  <h2 class = 'text-xl mx-2'>></h2>
+</div>
+<%= form_with model:@entity, url: group_entities_path(@group_id), class: 'mx-auto' do |form| %>
+  <div class="py-8 px-5 md:px-10 bg-white shadow-md rounded border border-gray-400">
+      <h1 class="text-center text-gray-800 font-lg font-bold tracking-normal leading-tight mb-4">Add new Entity</h1>
```

Comment:
- [ ] Same here, These headers suppose to be `Transactions` and `Add a new transaction` instead of `Entities` and `Add new Entity`, please fix that.
- [ ] It seems that you used two `<h1>` tags here on the same page Kindly change the second one to `h2` or `h3` for better semantics.

Reply:
Thanks @Omar-Muhamad  i am changing to match requeriments.
Also changed `h1` to `h2`